canonical-websites/www.ubuntu.com
# Moved data-privacy-enquiry into data-privacy area

## Done

- Updated legal data-privacy-enquiry section, moved it into the data-privacy folder
- Updated navigation
- Updated redirects
- Added some special redirects requested by legal team

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- look at the files in the /legal directory and compare to this [spreadsheet](https://docs.google.com/spreadsheets/d/12kuuGgwRqN3k7wlRc4QKIuZifjvxbynizO_LEjKjBq4/edit#gid=0)
* test the link [/legal/data-privacy/enquiry](http://0.0.0.0:8001/legal/data-privacy/enquiry) and [the old link](http://0.0.0.0:8001/legal/data-privacy-enquiry) redirects

## Issue / Card

Fixes #4193
